README.md

```diff
@@ -289,6 +289,18 @@ git branch -d feature/AmazingFeature
 ```
 
 ### Deployments
+
+To start a production deployment [create a PR from `develop` to `main`](https://github.com/djangonaut-space/wagtail-indymeet/compare/main...develop?title=Production%20Release%20-%20&body=PRs:%0A-%20) (bookmark this link for quick creation of PRs). The PR should follow this format:
+
+```
+Title: "Production release - <summary>"
+
+Description:
+PRs:
+- #1
+- #2
+```
+
```

Comment:
Shall we add that you merge with a merge commit? Whereas to develop we squash merge?

Reply:
I'm fine adding the mention about a merge commit here. Less so about develop since that may vary.